internal/common/db.go

```diff
@@ -37,6 +37,15 @@ type Cluster struct {
 	AmsSubscriptionID strfmt.UUID `json:"ams_subscription_id"`
 }
 
+type Event struct {
+	gorm.Model
+	models.Event
+}
+
+func PrepareAssistedTables(db *gorm.DB) error {
```

Comment:
Can this just be called `AutoMigrate`?
`PrepareAssistedTables` feels like it should be doing more.

Also ... what other tables would we be preparing in assisted-service? :trollface:

Reply:
I didn't want to use `AutoMigrate` because it hadn't felt so self explanatory at first. But maybe whoever is experience with this operation should understand it.

And yeah we want to add some more. Starting with operators.